configs/indexes.json

```
@@ -1,480 +1,662 @@
+      "type": "http",
+      "format": "jsonPath",
+      "jsonPath": "$[0][4]"
+    }
+  ],
+  "XAU/USD": [
+    {
+      "URL": "https://forex-data-feed.swissquote.com/public-quotes/bboquotes/instrument/XAU/USD",
+      "type": "http",
+      "format": "jsonPath",
+      "jsonPath": "$[0].spreadProfilePrices[0].ask"
+    }
+  ],
+  "XFT/USD": [
+    {
+      "URL": "https://api.coingecko.com/api/v3/simple/price?ids=offshift&vs_currencies=usd",
+      "type": "http",
+      "format": "jsonPath",
+      "jsonPath": "$.offshift.usd"
+    }
+  ],
+  "XLM/BTC": [
+    {
+      "URL": "https://api.binance.com/api/v1/klines?symbol=XLMBTC&interval=1d&limit=1",
+      "type": "http",
+      "format": "jsonPath",
+      "jsonPath": "$[0][4]"
+    }
+  ],
+  "XMR/ETH": [
+    {
+      "URL": "https://api.binance.com/api/v1/klines?symbol=XMRETH&interval=1d&limit=1",
+      "type": "http",
+      "format": "jsonPath",
+      "jsonPath": "$[0][4]"
+    }
+  ],
+  "XMR/USD": [
+    {
+      "URL": "https://api.coingecko.com/api/v3/simple/price?ids=monero&vs_currencies=usd",
+      "type": "http",
+      "format": "jsonPath",
+      "jsonPath": "$.monero.usd"
+    }
+  ],
+  "XRP/BTC": [
+    {
+      "URL": "https://api.binance.com/api/v1/klines?symbol=XRPBTC&interval=1d&limit=1",
+      "type": "http",
+      "format": "jsonPath",
+      "jsonPath": "$[0][4]"
+    }
+  ],
+  "XRP/USD": [
+    {
+      "URL": "https://api.binance.com/api/v1/klines?symbol=XRPUSDT&interval=1d&limit=1",
+      "type": "http",
+      "format": "jsonPath",
+      "jsonPath": "$[0][4]"
+    },
+    {
+      "URL": "https://api.kraken.com/0/public/Ticker?pair=XRPUSD",
+      "type": "http",
+      "format": "jsonPath",
+      "jsonPath": "$.result.XXRPZUSD.c[0]"
+    },
+    {
+      "URL": "https://api.coingecko.com/api/v3/simple/price?ids=ripple&vs_currencies=usd",
+      "type": "http",
+      "format": "jsonPath",
+      "jsonPath": "$.ripple.usd"
+    }
+  ],
+  "XTZ/USD": [
+    {
+      "URL": "https://api.coingecko.com/api/v3/simple/price?ids=tezos&vs_currencies=usd",
+      "type": "http",
+      "format": "jsonPath",
+      "jsonPath": "$.tezos.usd"
+    }
+  ],
+  "ZEC/ETH": [
+    {
+      "URL": "https://api.binance.com/api/v1/klines?symbol=ZECETH&interval=1d&limit=1",
+      "type": "http",
+      "format": "jsonPath",
+      "jsonPath": "$[0][4]"
+    }
+  ],
+  "ZEC/USD": [
+    {
+      "URL": "https://api.binance.com/api/v1/klines?symbol=ZECUSDC&interval=1d&limit=1",
+      "type": "http",
+      "format": "jsonPath",
+      "jsonPath": "$[0][4]"
+    }
+  ],
+  "ZRX/BNB": [
+    {
+      "URL": "https://api.binance.com/api/v1/klines?symbol=ZRXBNB&interval=1d&limit=1",
+      "type": "http",
+      "format": "jsonPath",
+      "jsonPath": "$[0][4]"
+    }
+  ],
+  "ZRX/USD": [
+    {
+      "URL": "https://api.binance.com/api/v1/klines?symbol=ZRXUSDT&interval=1d&limit=1",
+      "type": "http",
+      "format": "jsonPath",
+      "jsonPath": "$[0][4]"
```

Now that I see it I think these are better names.
```suggestion
      "parser": "jsonPath"
      "param": "$[0][4]"
```

The same field will be used for all parsers so it needs to be more generic.
for example, imagine the following combinations
```
parser: XML
param: x.v.b

parser: jsonPath
param: $[0][4]
```

So essentially this becomes the parser parameter.